hackathon_site/dashboard/frontend/yarn.lock

```diff
@@ -8120,6 +8120,18 @@ memory-fs@^0.5.0:
     errno "^0.1.3"
     readable-stream "^2.0.1"
 
+<<<<<<< HEAD
```

Comment:
Uhh this isn't good, this is a merge conflict in yarn.lock. Were you doing something in the react site that generated this, or did this happen when resolving conflicts?

Since there aren't any react changes here, you should check this back out from develop:

```bash
git checkout develop hackathon_site/dashboard/frontend/yarn.lock
```

Reply:
got it! When I was trying to merge develop to IEEE-3 branch, yarn.lock was not merged.